tests/conftest.py

```diff
@@ -1,5 +1,7 @@
 import pytest
 import trustme
+from cryptography.hazmat.backends import default_backend
```

Comment:
I'm sorry, but I missed where this dependency is specified?

May be I'm wrong, but using a dependency in import section, you should specify it somewhere, e.g. in either `requirements.txt` or  `setup.py`

Reply:
you're absolutely right, but I think trustme installs it by default, will check to be 100% sure

Reply:
yep we got covered by it:

https://github.com/python-trio/trustme/blob/a09f8e2ccc2ff321c4aee7870077bffb998d73f0/setup.py#L17-L23

Reply:
If you asked me, I'd add it to the requirements because you directly import this library in the project source.

The logic is, having omitted `cryptography` for now, you create an implicit dependency on `trustme` code (a-la "`trustme` provides us with `cryptography`).

If it wasn't "test" code, I'd say that it's mandatory to add `cryptography` to the `uvicorn` dependencies, but may be it's acceptible for test code

Reply:
OK I added it, I agree with you, better be explicit, you changed my mind !

Reply:
Awesome! Feel free to merge it 🚀